auto_nag/bzcleaner.py

```diff
@@ -127,6 +136,12 @@ def all_include_fields(self):
     def get_max_ni(self):
         return -1
 
+    def get_max_actions(self):
+        return -1
+
+    def exclude_no_action_bugs(self):
+        return True
```

Comment:
What is the need for this?

Reply:
I was thinking that in some cases, it could be helpful to get a complete list of bugs, including the ones we did not perform an action on them for exceeding the limit. We could highlight in the email that we did not act to avoid exceeding the limit.

Reply:
OK, makes sense, can you add a comment?

Reply:
Sure, I added a comment in bfe9aae95ed600be3fb6206f6a26b0c27a8dc9aa.